### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
- Resolves https://github.com/AgoraIO/Docs-Source/issues/467
- Resolves https://github.com/AgoraIO/Docs-Source/issues/468
- Resolves https://github.com/AgoraIO/Docs-Source/issues/499
- Resolves https://github.com/AgoraIO/Docs-Source/issues/500
- Resolves https://github.com/AgoraIO/Docs-Source/issues/501
- Resolves https://github.com/AgoraIO/Docs-Source/issues/502
- Resolves https://github.com/AgoraIO/Docs-Source/issues/503
- Resolves https://github.com/AgoraIO/Docs-Source/issues/504
- Resolves https://github.com/AgoraIO/Docs-Source/issues/505

### CODE REVIEW

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there.

File: general-product-inquiry/agora_product.mdx [hunk 1]
```diff
@@ -9,9 +9,16 @@ export const toc = [{}];
 
 Agora is a Platform as a Service (PaaS) provider that enables real-time communicatons with easy-to-embed SDKs and RESTful APIs. Through calling the APIs, you can add the following functions in your applications:
 
-- [Voice Call](../../voice-calling/overview/product-overview)
-- [Video Call](../../video-calling/overview/product-overview)
-- [Live Interactive Streaming](../../interactive-live-streaming/overview/product-overview)
-- [Cloud Recording](../../cloud-recording/overview/product-overview)
-- [On-premise Recording](../../on-premise-recording/overview/product-overview)
-- [Real-time Messaging](../../signaling/overview/product-overview)
+- [Video Calling](https://docs.agora.io/en/video-calling/overview/product-overview)
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply general-product-inquiry/agora_product.mdx]: Resolved

File: general-product-inquiry/capacity.mdx [hunk 2]
```diff
@@ -13,26 +13,6 @@ In most scenarios, Agora recommends limiting the number of users sending streams
...
-    </a>
-    . If you want to use the 128-host feature, contact <a href="mailto:support@agora.io">
-      support@agora.io
-    </a> to help with the implementation.{' '}
-  </li>
-</div>
+- If the number of users sending streams concurrently exceeds the recommended limit, each user in the channel only sees or hears streams equal to the recommended limit, and these streams are assigned at random for each user. For example, if 18 hosts are sending streams concurrently in a live streaming channel, each user sees or hears 17 of the 18 streams, and the missing stream is random.
+- Agora does not provide APIs for limiting the number of users sending streams concurrently. However, you can implement such a limitation in your application layer.
+- To minimize bandwidth consumption and ensure smooth communication in a real-time video streaming scenario with multiple users, Agora recommends referring to <a href="https://docs.agora.io/en/Interactive%20Broadcast/multi_user_video?platform=All%20Platforms">Video for Multiple Users</a>. If you want to use the 128-host feature, contact <a href="mailto:support@agora.io">support@agora.io</a> to help with the implementation.{' '}
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply general-product-inquiry/capacity.mdx]: Used the multi channel doc relative instead now

File: integration-issues/string.mdx [hunk 3]
```diff
@@ -5,33 +5,11 @@ platforms: []
...
-    </li>
-    <li>
-      <a href="../../video-calling/reference/agora-console-rest-api">
-        Console RESTful API
-      </a>
-    </li>
-  </p>
-</div>
+This feature is in BETA. We recommend contacting support@agora.io before implementing this function.<p>The following products or features do not support string user accounts:
+- <a href="https://docs.agora.io/en/cloud-recording/product_cloud_recording?platform=All%20Platforms">Cloud Recording</a >
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply integration-issues/string.mdx]: Fixed

**Comment by billy-the-fish** [comment]
@Kishan-Dhakan , can you quickly check the links in https://docs.agora.io/en/help/quality-issues/video_blank and https://docs.agora.io/en/help/general-product-inquiry/browser_support please. If they are not good, can you update please. Plus go through https://docs.google.com/spreadsheets/d/1kGLHc-FkXO_paRkFQA6BlawKw3zqtAdzUC55sahR-AI/edit#gid=1874725026 and fix all links. Thanks in advance.

File: integration-issues/string.mdx [hunk 4]
```diff
@@ -5,33 +5,11 @@ platforms: []
...
-        Console RESTful API
-      </a>
-    </li>
-  </p>
-</div>
+This feature is in BETA. We recommend contacting support@agora.io before implementing this function. The following products or features do not support string user accounts:
+- [Cloud Recording](/cloud-recording/overview/product-overview)
+- <a href="/en/3.x/interactive-live-streaming/advanced-features/media-push#api-reference?platform=android">RTMP Converter</a >
+- <a href="/en/3.x/video-calling/reference/channel-management-rest-api">Channel Management RESTful API</a >
+- <a href="/en/3.x/video-calling/reference/agora-console-rest-api">Console RESTful API</a >
```
Comment (Kishan-Dhakan) [inline integration-issues/string.mdx:12]: Works

**Review by Kishan-Dhakan — COMMENTED** [review]
Links checked

**Review by billy-the-fish — APPROVED** [review]
LGTM.

**Pull request merged**